metaseq/checkpoint_utils.py

```diff
@@ -179,18 +179,19 @@ def _delete_old_checkpoint_files(
                 os.remove(old_chk)
 
 
-def verify_shards(cfg, dir=None, checkpoint_name=None):
+def verify_shards(cfg, dir=None, checkpoint_name=None, distributed_world_size=None):
```

Comment:
This seems to be from another PR. May be remove the changes from this file

Reply:
done